src/Http/Request.php

```diff
@@ -283,6 +303,16 @@ protected static function parseBody($method) {
   }
 
   /**
+   * Parses the input data.
+   *
+   * @return array
+   *   The parsed input.
+   */
+  protected static function parseInput($method) {
+    return $_GET;
```

Comment:
I think it is not. This returns an structured array of the user input via query string.